rest/src/main/java/org/pdxfinder/controllers/MappingsController.java

```diff
@@ -157,17 +157,10 @@ public MappingsController(
     return new ResponseEntity<>(responseBody, responseStatus);
   }
 
-//    @GetMapping("/mappings/ontologies")
-//    public Object getOntologies(@RequestParam(value = "type", defaultValue = "diagnosis") Optional<String> dataType){
-//
-//        String entityType = dataType.get();
-//        return mappingService.getOntologyTermsByType(entityType);
-//    }
-
-
   @GetMapping("export")
   @ResponseBody
-  public Object exportMappingData(HttpServletResponse response,
+  public Object exportMappingData(
+      HttpServletResponse response,
       @RequestParam("mq") Optional<String> mappingQuery,
       @RequestParam(value = "mapped-term", defaultValue = "") Optional<String> mappedTermLabel,
```

Comment:
Any idea why Optionals were used?

Reply:
Not really... Maybe they are not necessary here as anyways a defaultValue is provided. I will check if removing them changes anything functionally.